components/Pagination/src/PaginationLink.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import clsx from 'clsx';
+
+interface Props {
+  page: number;
+  index: number;
+  currentIndex: number;
+  onClick: (index: number) => void;
+}
+
+export const PaginationLink = ({ page, index, currentIndex, onClick }: Props) => {
```

Comment:
Om verwarring te voorkomen zou deze gewoon noemen naar wat het is, dus een PaginationButton.